docs/scos/user/features/202108.0/promotions-discounts-feature-overview.md

```diff
@@ -125,93 +125,258 @@ There are two types of discount application:
 
 ## Discount validity interval
 
 Validity interval is a time period during which a discount is active and can be applied.
 
 If a cart is eligible for a discount outside of its validity interval, the cart rule is not applied. If a customer enters a voucher code outside of its validity interval, they get a "Your voucher code is invalid." message.
 
-A product catalog manager defines calculation when [creating a voucher](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html) or [creating a cart rule](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-cart-rules.html).
+A Back Office user defines calculation when [creating a voucher](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html) or [creating a cart rule](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-cart-rules.html).
 ![Validity interval](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Promotions+&+Discounts/Discount/Discount+Feature+Overview/validity-interval.png)
 
+## Discount priority
+
+In cases when several discounts can apply to a customer order, a Back Office user can set the discount priorities. The discount priority is an integer value that defines the order in which the discounts are applied. See [Discount calculation logic](#discount-calculation-logic) for details on how discounts are calculated based on their priorities.
+
+Discount prioritization behavior does not depend on the [discount type](#discount-types). For example, you might have the following priorities for different types:
+
+* Cart rule discount with priority 1
+* Voucher code discount with priority 2
+* Voucher code discount with priority 3
+* Cart rule discount with priority 4
+
+In this case, the discounts are applied in the order of priority, irrespective of the type.
+
+{% info_block infoBox "Info" %}
+
+If you have a voucher code and a cart rule with the same priority, then both discounts should apply and use the same cart subtotal. See [Scenario 5: Non-exclusive, fixed amount and percentage discounts with the same priority values](#scenario-5-non-exclusive-fixed-amount-and-percentage-discounts-with-the-same-priority-values) for an example.
+
+{% endinfo_block %} 
+
+## Discount calculation logic
+
+The discount calculation logic follows these rules:
+
+* All discounts are calculated sequentially based on their [priority value](#discount-priority). Thus, discounts with the highest priority value are considered first.
+* Discounts without priority values are treated with the last possible priority.
+* Discounts with the same priority value are calculated independent of each other.
```

Comment:
```suggestion
* Discounts with the same priority value are calculated independently.
```

Reply:
To avoid confusion, I wanted to emphasize that they are calculated independent on each other